src/MultipleComplaintImageFactory.js

```diff
@@ -203,6 +222,26 @@ export default function(Complaints, SelectImages) {
       let image = this.refs.image;
       image.width(width - imagePadding);
       image.height(width - imagePadding);
+
+      let { active } = this.state;
+      let complaints = this.state[active] ? this.state[active].complaints : [];
+      let srcWidth =
+        this.state.srcWidth !== undefined ? this.state.srcWidth : "";
+
+      if (
+        srcWidth !== "" &&
+        this.currentWidth !== srcWidth &&
+        complaints.length > 0
+      ) {
+        complaints.forEach(item => {
+          let positionX = "";
+          let positionY = "";
+          positionX = item.pos.x * (this.currentWidth / srcWidth);
+          positionY = item.pos.y * (this.currentWidth / srcWidth);
```

Comment:
This logic assumes the image is always gonna have the same width and height. Is the true? Have you tried using multiple images with different dimensions?

Reply:
The width changes according to the resolutions. Tried with different images with different dimensions all looks good.

Based on the current width and previously stored width the x and y positions are placed.